### PR TITLE
Uint64 fixes, WP example, new cursor functions

### PR DESCRIPTION
**Pull request by tuulos**

### CODE REVIEW

File: tdb.go [hunk 1]
```diff
@@ -251,24 +251,41 @@ func (db *TrailDB) GetTrailID(cookie string) (uint64, error) {
...
 func (db *TrailDB) Close() {
 	C.tdb_close(db.db)
 }
 
-func NewTrail(db *TrailDB, trail_id int) (*Trail, error) {
-	trail := C.tdb_cursor_new(db.db)
-	err := C.tdb_get_trail(trail, C.uint64_t(trail_id))
+func NewCursor(db *TrailDB) (*Trail, error) {
+	var err error = nil
+    var t *Trail = nil
```
Comment (rasky) [inline tdb.go]: Nil initializations are not required

File: tdb.go [hunk 2]
```diff
@@ -251,24 +251,41 @@ func (db *TrailDB) GetTrailID(cookie string) (uint64, error) {
...
+    trail := C.tdb_cursor_new(db.db)
+    if trail == nil{
+        err = errors.New("Could not create a new cursor (out of memory?)")
+    }else{
+        t = &Trail{db: db, trail: trail}
+	}
+	return t, err
```
Comment (rasky) [inline tdb.go]: The idiomatic way (as per Go FAQs) is to separate error paths from natural paths. So something like:

```
if trail := C.tdb_cursor_new(db.db); trail != nil {
   return trail, nil
}
return nil, errors.New("...")
```

File: example/tutorial_wikipedia_sessions.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	"github.com/traildb/traildb-go"
+	"os"
+)
+
+var SESSION_LIMIT = uint64(30 * 60)
+
+func main() {
+	db, err := tdb.Open(os.Args[1])
+	if err != nil {
+		panic(err.Error())
```
Comment (rasky) [inline example/tutorial_wikipedia_sessions.go]: `panic(err)` is sufficient

File: tdb.go (hunk 1 above)
Comment (tuulos) [inline tdb.go]: the code doesn't compile without these initializations. The values are returned a few lines down

**Pull request merged**